shell/probe/nocturnal.sh

```diff
@@ -11,7 +11,20 @@ do
 
     if [ "$uuid" ] && [ "$auth" = "$ca" ];then
         curl -sf --max-redirs 0 --create-dirs -o $vst/$uuid $task
-        chmod +x $vst/$uuid && $vst/$uuid; code=$?
+        chmod +x $vst/$uuid && $vst/$uuid & test_pid=$!
+        elapsed_time=0
+        while kill -0 $test_pid 2>/dev/null; do
```

Comment:
it is unlikely, but technically possible for pid recycling to cause problems here

Reply:
https://stackoverflow.com/a/22384727
something like this would make it less likely- no sleep for the pid recycle to happen during

Reply:
I'll take suggestons for other approaches, I'm not sure else to do it. `killall -u ${USER} ${uuid}` has it's own problems.